docs/getting_started.md

```diff
@@ -216,6 +218,64 @@ Here are some resources that can be used to get you started working with `AnnDat
 - [Converting directly from R to Python](https://theislab.github.io/scanpy-in-R/#converting-from-r-to-python)
 - [Homepage for scanpy tutorials](https://scanpy.readthedocs.io/en/latest/tutorials.html)
 
+
+## Special considerations for CITE-seq experiments
+
+If the dataset that you have downloaded contains samples with ADT data from a CITE-seq experiment, the raw and normalized ADT expression matrices are stored in an `altExp` named `"adt"` of the `SingleCellExperiment` object in the `processed.rds` file:
+
+```r
+# View the ADT alternative experiment
+altExp(processed_sce)
+
+# You can also explicitly specify the altexp's name:
+altExp(processed_sce, "adt")
+```
+
+The following commands can be used to access the ADT expression matrices:
+
+```r
+# the raw ADT counts matrix
+raw_adt_counts <- counts(altExp(processed_sce))
+
+# the log normalized ADT counts matrix
+normalized_adt_counts <- logcounts(altExp(processed_sce))
+```
+
+Be aware that the `SingleCellExperiment` object in the `processed.rds` file was not filtered based on ADT counts.
```

Comment:
```suggestion
Be aware that the `SingleCellExperiment` object in the `processed.rds` file has been filtered to remove low-quality cells based on RNA expression and is not filtered based on ADT counts.
```
I think we should be clear about what is in the processed object and that there is filtering, just not on the ADT matrix.